docs/get-ckb.md

```diff
@@ -1,9 +1,34 @@
 # Get CKB
 
+We have CI to ensure CKB works in following systems, they are also the
+recommended system to run CKB.
+
+OS           | Arch
+-------------|------
+Ubuntu 16.04 | x64
+macOS        | x64
+Windows\*    | x64
+
+> \* Support for Windows is experimental. CKB is known to have serious performance
+> issues when running in Windows.
+
+CKB should also work on any modern Linux distributions. However, we limit our
+energy to focus on the support of following systems:
+
+OS             | Arch
+---------------|------
+Ubuntu 18.04   | x64
+Debian Stretch | x64
```

Comment:
Planned release date of `Debian Buster` is [2019-07-06](https://lists.debian.org/debian-devel-announce/2019/06/msg00003.html).

I will push request to change this in July, 2019.